tests/tracer/test_encoders.py

```diff
@@ -795,139 +792,3 @@ def test_json_encoder_traces_bytes():
         assert "\ufffdspan.a" == span_a["name"], span_a["name"]
         assert "\x80span.b" == span_b["name"]
         assert "\ufffdspan.b" == span_c["name"]
-
-
-@pytest.mark.skipif(
-    os.getenv("PYTHONOPTIMIZE", "").lower() in ("1", "t", "true"),
-    reason="Python optimize removes assertions from cython code",
-)
-def test_verifying_v05_payloads():
```

Comment:
Why is this test being dropped?

Reply:
All this PR does is revert the code that was introduced in this PR and resolves some linting conflicts. There are no additional changes: https://github.com/DataDog/dd-trace-py/pull/7277.